docs/core/install/localized-intellisense.md

```diff
@@ -70,7 +69,7 @@ For Visual Studio to use a different language for IntelliSense, install the appr
 If you didn't install the desired language pack during setup, update Visual Studio as follows to install the language pack:
 
 > [!IMPORTANT]
-> To install, update, or modify Visual Studio, you must log on with an account that has administrative permissions. For more information, see [User permissions and Visual Studio](/visualstudio/ide/user-permissions-and-visual-studio).
+> To install, update, or modify Visual Studio, you must log on with an account that has administration permission. For more information, see [User permissions and Visual Studio](/visualstudio/ide/user-permissions-and-visual-studio).
```

Comment:
nit: Just to be consistent with another sentence in the same article:

> This procedure requires that you have **administrator permission** to copy the IntelliSense files to the .NET Core installation folder.

Reply:
so can you use a similar sentence as used in the other sentence in this article?

Reply:
```suggestion
> To install, update, or modify Visual Studio, you must log on with an account that has administrator permission. For more information, see [User permissions and Visual Studio](/visualstudio/ide/user-permissions-and-visual-studio).
```